public/app/features/playlist/PlaylistEditPage.test.tsx

```diff
@@ -76,6 +76,7 @@ describe('PlaylistEditPage', () => {
       fireEvent.submit(screen.getByRole('button', { name: /save/i }));
       await waitFor(() => expect(putMock).toHaveBeenCalledTimes(1));
       expect(putMock).toHaveBeenCalledWith('/api/playlists/foo', {
+        uid: 'foo',
```

Comment:
I changed the update command to use the uid in the payload rather than passing it as a sibling argument, so these test snapshots need updated